### PR TITLE
Set ETCD_UNSUPPORTED_ARCH on arm64

### PR DESCRIPTION
**Pull request by yselkowitz**

### CODE REVIEW

File: pkg/cmd/render/env.go [hunk 1]
```diff
@@ -82,7 +81,10 @@ func getElectionTimeout(platform, arch string) (map[string]string, error) {
 }
 
 func getUnsupportedArch(platform, arch string) (map[string]string, error) {
-	if !strings.HasPrefix(arch, "s390") {
+	switch arch {
+	case "arm64":
+	case "s390x":
```
Comment (hexfusion) [inline pkg/cmd/render/env.go:86]: we need to cover s390 case as well as s390x

**Comment by hexfusion** [comment]
@yselkowitz thanks for PR small knit above.

**Comment by hexfusion** [comment]
can you outline the support for arm?
/approved

File: pkg/cmd/render/env.go (hunk 1 above)
Comment (yselkowitz) [reply pkg/cmd/render/env.go:86]: Why?  Plain s390 means the very old 31-bit userspace, for which RHEL has only had multilibs in previous versions and not at all in RHEL 8.  There are certainly no plans for OCP in that configuration.

Comment (hexfusion) [reply pkg/cmd/render/env.go:86]: ok I have no context here to what s390 is outside of "both were possible" if you're saying that is not true and regression is not a concern then OK with me.

**Comment by hexfusion** [comment]
do we have any plans for e2e testing to include arm base infra?

**Pull request merged**